app/controllers/api/v1/invoice_items/random_controller.rb

```diff
@@ -1,8 +1,7 @@
 class Api::V1::InvoiceItems::RandomController < ApplicationController
-  respond_to :json, :xml
 
   def show
-    respond_with InvoiceItem.limit(1).order("RANDOM()")
+    render json: InvoiceItem.limit(1).order("RANDOM()")
```

Comment:
Do you know of a better way to do this than `limit(1)`? I tried using `take` but it didn't work. Threw an error